### DISCUSSION TITLE
How to set default folder or...

### DISCUSSION THREAD

**Discussion by cedardocster** [Zettlr/Zettlr#4207] [General]
How do you set a default folder for Zettlr (on debian version), or at least open Zettlr with a specific project folder from the terminal without having to run Zettlr then manually pick the project from the menus?

Problem:  if I do this

Zettlr "/home/david/SyncthingFolders/ZettlrMarkor/FrontPage.md" 

from the command line it does open Zettlr but the page doesn't open until you manually pick the right project folder

actually, even if the project folder is opened it still doesn't load it from the command line :(

Regardless, I'd still like to be able to open Zettlr in the project folder I specify (please!)

**Comment by nathanlesage** [comment]
This sounds like the idea of more detached workspaces as VSCode also has them and as Frank has asked for recently over on Discord. But it also sounds like a small bug? It might help if you could give more info about what you do, what you expect, where Zettlr does something weird, and detach that from something that you would also see. This would help to separate bugs from feature requests!

**Reply by cedardocster** [reply]
david@hpb:~$ Zettlr "/home/david/SyncthingFolders/zettlrEpsilon/FrontPage.md"
[Application] Performing environment check ...
[Application] Pandoc has been bundled with this release. Path: /opt/Zettlr/resources/pandoc
[Application] Environment check complete.
(node:319251) Warning: Keys with collection values will be stringified due to JS Object restrictions: "{ date }". Set mapAsMap: true to use object keys.
(Use `Zettlr --trace-warnings ...` to show where the warning was created)
(node:319251) Warning: Keys with collection values will be stringified due to JS Object restrictions: "{ date }". Set mapAsMap: true to use object keys.


about 45 seconds later the Zettlr window opens but not at the right page (but it is in the right folder)

I right click this file in the file manager (Thunar) and select to open with Zettlr and get this error:
	A JavaScript error occured in the main process
	Uncught Exception:
	TypeError: c.windows.showMainWindows is not a function
	at App.<anonymous> (/opt/Zettlr/resources/app.asar/.webpack/main/index.js:2:2044791)
	at App.emit (node:events:513:28)


at a new terminal I do this:

Zettlr "/home/david/SyncthingFolders/zettlrEpsilon/Zettlr notes from terminal.md"

and get the same error

I do this 
Zettlr "/home/david/SyncthingFolders/temp/zettlrEpsilon/FrontPage.md"
 in a new terminal and get the same 45 second delay as before, the previous Zettlr window is closed and a new one opens, still at the wrong page
 
using Zettlr 3.0.0-beta.2.nightly, but I'm pretty sure I get the same thing happening with the stable release (2.3.something)
